chaos/litmus/master.yaml

```diff
@@ -0,0 +1,66 @@
+---
+# mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
+# fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
+# Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
+# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
+# will help with categorization & application-mapping
+
+
+litmus_test_1:
```

Comment:
I think we should remove this as a key instead we should have list of tests. What do you think?

like:
```
- name: pod_kill
  ...
- name: node_taint
  ...
```

Reply:
Yes @a4abhishek . That is what we will replace it with once we have the explicit chaos tests ready. This is just a reference/placeholder.